<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>信号发生-传输-接收分析简易系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="generatorTab">
       <attribute name="title">
        <string>信号发生器</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGroupBox" name="signalSettingsGroupBox">
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>信号设置</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="signalTypeLabel">
             <property name="text">
              <string>信号类型:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="signalTypeComboBox">
             <item>
              <property name="text">
               <string>正弦波</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>方波</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>三角波</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>文件数据</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="frequencyLabel">
             <property name="text">
              <string>频率(Hz):</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QDoubleSpinBox" name="frequencySpinBox">
             <property name="minimum">
              <double>0.100000000000000</double>
             </property>
             <property name="maximum">
              <double>1000.000000000000000</double>
             </property>
             <property name="value">
              <double>100.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="amplitudeLabel">
             <property name="text">
              <string>幅度:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QDoubleSpinBox" name="amplitudeSpinBox">
             <property name="minimum">
              <double>0.100000000000000</double>
             </property>
             <property name="maximum">
              <double>32767.000000000000000</double>
             </property>
             <property name="value">
              <double>1000.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="dcOffsetLabel">
             <property name="text">
              <string>直流偏置:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QDoubleSpinBox" name="dcOffsetSpinBox">
             <property name="minimum">
              <double>-32768.000000000000000</double>
             </property>
             <property name="maximum">
              <double>32767.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="samplingRateLabel">
             <property name="text">
              <string>采样率:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QComboBox" name="samplingRateComboBox">
             <item>
              <property name="text">
               <string>1 KHz</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2 KHz</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4 KHz</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>8 KHz</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="filePathLabel">
             <property name="text">
              <string>数据文件:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLineEdit" name="filePathEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QPushButton" name="loadFileButton">
             <property name="text">
              <string>加载文件</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0" colspan="2">
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="8" column="0" colspan="2">
            <widget class="QPushButton" name="startGeneratorButton">
             <property name="text">
              <string>开始生成信号</string>
             </property>
            </widget>
           </item>
           <item row="9" column="0" colspan="2">
            <widget class="QPushButton" name="stopGeneratorButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>停止生成信号</string>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="logLabel">
             <property name="text">
              <string>日志:</string>
             </property>
            </widget>
           </item>
           <item row="11" column="0" colspan="2">
            <widget class="QTextEdit" name="logTextEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="generatorChartLayout"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="channelTab">
       <attribute name="title">
        <string>信道模块</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QGroupBox" name="channelSettingsGroupBox">
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>信道设置</string>
          </property>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="noiseLabel">
             <property name="text">
              <string>噪声幅度:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="noiseAmplitudeLabel">
             <property name="text">
              <string>1.0</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QSlider" name="noiseAmplitudeSlider">
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="value">
              <number>10</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="channelChartLayout"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="receiverTab">
       <attribute name="title">
        <string>接收分析</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QSplitter" name="receiverSplitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QGroupBox" name="receiverSettingsGroupBox">
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="title">
            <string>接收设置</string>
           </property>
           <layout class="QFormLayout" name="formLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="filterCutoffLabel">
              <property name="text">
               <string>滤波截止频率:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDoubleSpinBox" name="filterCutoffSpinBox">
              <property name="minimum">
               <double>10.000000000000000</double>
              </property>
              <property name="maximum">
               <double>4000.000000000000000</double>
              </property>
              <property name="value">
               <double>500.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QPushButton" name="saveDataButton">
              <property name="text">
               <string>保存数据到文件</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QSplitter" name="chartSplitter">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QWidget" name="receiverChartWidget" native="true">
            <layout class="QVBoxLayout" name="receiverChartLayout"/>
           </widget>
           <widget class="QWidget" name="spectrumChartWidget" native="true">
            <layout class="QVBoxLayout" name="spectrumChartLayout"/>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="oscilloscopeTab">
       <attribute name="title">
        <string>示波器</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QGroupBox" name="oscilloscopeSettingsGroupBox">
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>示波器设置</string>
          </property>
          <layout class="QFormLayout" name="formLayout_4">
           <item row="0" column="0">
            <widget class="QLabel" name="timePerDivLabel">
             <property name="text">
              <string>时间/格(秒):</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QDoubleSpinBox" name="timePerDivSpinBox">
             <property name="minimum">
              <double>0.000000000000000</double>
             </property>
             <property name="maximum">
              <double>10.000000000000000</double>
             </property>
             <property name="value">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="voltPerDivLabel">
             <property name="text">
              <string>电压/格:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QDoubleSpinBox" name="voltPerDivSpinBox">
             <property name="minimum">
              <double>0.100000000000000</double>
             </property>
             <property name="maximum">
              <double>10000.000000000000000</double>
             </property>
             <property name="value">
              <double>5000.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="triggerModeLabel">
             <property name="text">
              <string>触发模式:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="triggerModeComboBox">
             <item>
              <property name="text">
               <string>自动</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>普通</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>单次</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="triggerLevelLabel">
             <property name="text">
              <string>触发电平:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QDoubleSpinBox" name="triggerLevelSpinBox">
             <property name="minimum">
              <double>-32768.000000000000000</double>
             </property>
             <property name="maximum">
              <double>32767.000000000000000</double>
             </property>
             <property name="value">
              <double>0.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="Line" name="line_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="5" column="0" colspan="2">
            <widget class="QPushButton" name="startOscilloscopeButton">
             <property name="text">
              <string>启动示波器</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0" colspan="2">
            <widget class="QPushButton" name="stopOscilloscopeButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>停止示波器</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="oscilloscopeChartLayout"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
